src/components/Subscription.tsx

```diff
@@ -81,6 +81,8 @@ export default function Subscription({
         if (!signer) return
         const receipt = await predictorInstance.buyAndStartSubscription(signer)
 
+        await sleep(3000)
```

Comment:
it looks like a bit long to me

Reply:
I'll try some smaller values

Reply:
reduced to 1 sec, if there are going to be issues we can increase